server_addon/maya/client/ayon_maya/api/plugin.py

```diff
@@ -35,6 +34,9 @@
 from .lib import imprint, read
 from .pipeline import containerise
 
+# Backwards compatibility
+NewCreator = Creator
```

Comment:
I think we can remove this. It's not imported anywhere.
```suggestion
```

Reply:
Sweet. I was just preparing for a studio maybe inheriting from it for whatever reason, but it's odd maybe if they were relying on importing this anyway.